images/Dockerfile.recovery-controller

```diff
@@ -0,0 +1,11 @@
+FROM golang:alpine AS build-env
+ADD ./cassandra-recovery /cassandra-recovery
+WORKDIR /cassandra-recovery
+RUN apk add --no-cache gcc musl-dev git mercurial ca-certificates
```

Comment:
Isn't that already installed as part of the [Docker image](https://github.com/docker-library/golang/blob/e7fb0efd448658b759c2958744a8c59b53dcd6b3/1.14/alpine3.11/Dockerfile)? Also, `CGO_ENABLED=0`, do we still need `gcc` in that case?

Reply:
Good point, nope :) Actually, only git needs to be installed, i've cleaned that up.